### PR TITLE
fix import __init__.py

### PR DESCRIPTION
**Pull request by SeRj-ThuramS**
fix import error

Traceback (most recent call last):
  File "/usr/local/bin/tufw", line 5, in <module>
    from tufw.__init__ import main
  File "/usr/local/lib/python3.10/dist-packages/tufw/__init__.py", line 7, in <module>
    from firewall import *
ModuleNotFoundError: No module named 'firewall'

### CODE REVIEW

File: src/tufw/__init__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 from sys import argv, executable
 
 from dialog import Dialog
-from firewall import *
+from tufw.firewall import *
```
Comment (a13ssandr0) [inline src/tufw/__init__.py:7]:
```suggestion
from .firewall import *
```

**Review by a13ssandr0 — CHANGES_REQUESTED** [review]
Thank you for the submission.
Code is working now.
Before merging I suggested to change the import of `tufw.firewall` to a relative one, I tested it with the module installed with pip and works.

**Comment by SeRj-ThuramS** [comment]
When the python module is installed as a standalone program on Linux, it
cannot find the file in the firewall.py package.  So I suggested fixing

пт, 10 мая 2024 г., 10:41 Alessandro Campolo ***@***.***>:

File: src/tufw/__init__.py (hunk 1 above)
Comment (phanirithvij) [reply src/tufw/__init__.py:7]: @SeRj-ThuramS can you approve this fix. It works for me with python 3.8.10 on ubuntu with the `.firewall` relative import

**Comment by a13ssandr0** [comment]
@SeRj-ThuramS @phanirithvij I changed the import to try both import methods when running as a pip module and when installed as a standalone program.
Hope it works for all now.

**Pull request closed without merging**